packages/fx-core/src/plugins/resource/bot/azureOps.ts

```diff
@@ -23,15 +23,16 @@ export class AzureOperations {
     botClient: AzureBotService,
     resourceGroup: string,
     botChannelRegistrationName: string,
-    msaAppId: string
+    msaAppId: string,
+    displayName?: string | undefined
```

Comment:
since you use `?:`, so you can remove `| undefined`;

Reply:
ok